test/api_test.py

```diff
@@ -19,11 +19,18 @@
 import logging
 import unittest
 
+import googleapiclient.errors
+import mock
+import nose.tools
+
 import hyou.api
 
 import http_mocks
```

Comment:
I saw three group of `import` statements. I think there should only be two: one for the imports from the standard library and another block for 3rd party libraries. But I might be wrong.